torch/_subclasses/fake_tensor.py

```diff
@@ -381,13 +381,33 @@ def from_real_tensor(
         if maybe_memo is not None:
             return maybe_memo
         # not yet supported in metatensors
+        # HANDLE deepcopy / real tensors safely
```

Comment:
I think we need to re-examine the original issue here - this change is preventing us from creating from_real_tensor for any non-meta tensors.  This is not really how we want to use `from_real_tensor` - we want to only error when we are calling `deepcopy()` on a tensor in FakeTensorMode that is not meta device

Reply:
In otherwords, I think this logic needs to move to intercept where `deepcopy()` gets called - perhaps we can breakpoint in the torch_dispatch to find where that is?